features/step_definitions/oauth.rb

```diff
@@ -1,18 +1,26 @@
-Given /^the #{QUOTED} oauth CRD is restored after scenario$/ do |name|
+# The resource is a "customresource" (CR), not "customresourcedefinition" (CRD). Correcting it and make existing use not broken
```

Comment:
The comment need update.

Reply:
I noticed, forgot to force push, now updated, help merge, thanks